src/main/java/Storage.java

```diff
@@ -0,0 +1,100 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
```

Comment:
I think it would be better to import the java.util packages separately so that we can easily see what dependencies this class has.